core/controllers/email_dashboard_test.py

```diff
@@ -719,11 +725,15 @@ def test_that_correct_emails_are_sent_to_max_n_recipients(self):
             # Check that emails are sent to max n qualified users.
             # One email is sent to submitter for query completion and second
             # is sent to one of the 2 qualified users.
-            messages = self.mail_stub.get_sent_messages()
-            self.assertEqual(len(messages), 2)
-            self.assertEqual(messages[0].to, self.SUBMITTER_EMAIL)
-            self.assertIn(
-                messages[1].to, [self.USER_A_EMAIL, self.USER_B_EMAIL])
+            messages = self._get_sent_email_messages(
+                to=self.SUBMITTER_EMAIL)
+            self.assertEqual(len(messages), 1)
+            self.assertEqual(messages[0].to, [self.SUBMITTER_EMAIL])
+            messages_a = self._get_sent_email_messages(
+                to=self.USER_A_EMAIL)
+            messages_b = self._get_sent_email_messages(
+                to=self.USER_B_EMAIL)
+            self.assertTrue((len(messages_a) == 1) or (len(messages_b) == 1))
```

Comment:
Maybe also check that the other one is 0?

You can verify sorted(len(messages_a), len(messages_b)) against [0, 1]